src/open_inwoner/configurations/bootstrap/siteconfig.py

```diff
@@ -12,13 +12,18 @@ class SiteConfigurationStep(BaseConfigurationStep):
 
     verbose_name = "Site configuration"
     required_settings = [
-        "SITE_NAME",
-        "SITE_SECONDARY_COLOR",
-        "SITE_ACCENT_COLOR",
+        "name",
+        "secondary_color",
+        "accent_color",
     ]
```

Comment:
The `required_settings` list should contain the names of the settings in `conf.settings`, otherwise `validate_requirements` wont work: https://github.com/maykinmedia/django-setup-configuration/blob/main/django_setup_configuration/configuration.py#L16-L31

You could maybe define the `setting_to_config` mapping on the class level and then in `is_configured` you could loop over the `required_settings` and grab the config field name from the mapping